.github/workflows/daily.yml

```diff
@@ -0,0 +1,33 @@
+name: "Daily tasks (cronjobs)"
```

Comment:
```suggestion
name: "Robotoff - Daily tasks (cronjobs)"
```

Reply:
This file can contain other daily tasks (not only PostgreSQL backup). I've added a better job name description for the DB backup.